jerry-core/include/jerryscript-debugger.h

```diff
@@ -28,12 +28,22 @@ extern "C"
  */
 
 /**
+ * Types for the client source wait and run method.
+ */
+typedef enum
+{
+  JERRY_DEBUGGER_WAIT_AND_RUN_FAIL = 0, /**< error occured while receive */
```

Comment:
/**< source is not received */
We can return early if we are in breakpoint mode, etc., so this would be clearer.